src/javascript/JContent/ContentRoute/ToolBar/Toolbar.test.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {shallow} from '@jahia/test-framework';
+import ToolBar from './ToolBar';
+import {useSelector} from 'react-redux';
+
+jest.mock('react-redux', () => ({
+    useDispatch: jest.fn(),
+    useSelector: jest.fn()
+}));
+
+jest.mock('~/JContent/JContent.redux', () => ({
+    cmClearSelection: jest.fn()
+}));
+
+jest.mock('connected-react-router', () => jest.fn(() => {}));
+
+describe('Toolbar', () => {
+    it('should not render multiple selection actions', () => {
+        useSelector.mockImplementation(() => ({
+            selection: []
+        }));
+
+        const toolbar = shallow(<ToolBar/>);
+        expect(toolbar.find('Typography').length).toEqual(0);
+        expect(toolbar.find('Button').length).toEqual(0);
+    });
+
+    it('should render multiple selection actions for 1 item', () => {
+        useSelector.mockImplementation(() => ({
+            selection: ['/test']
+        }));
+
+        const toolbar = shallow(<ToolBar/>);
+        expect(toolbar.find('Typography').prop('children')).toContain('itemsSelected');
```

Comment:
I would mock the "translate" function  and check it has been called with the expected parameters (label key and number of items) instead.

Reply:
ok, will see how to do that

Reply:
couldn't mock it, as soon as I try to mock react-i18next, it's breaks all my tests, due to some buttons coming from the old design system

Reply:
Ok, fine. It was just a suggestion.